### ISSUE TITLE
output of arcface loss function different to mxnet original Implementation

### ISSUE THREAD

**Issue by williamCorrea85** [wujiyang/Face_Pytorch#32]
Hello,
I tested the output of your arcface loss function and compared it to the original implementation of Insightface (mxnet). I gave them the loss weights as a parameter. Unfortunately the output obtained is not the same for both functions. You can find the script for comparing both functions in the following link: https://github.com/williamCorrea85/arcface/blob/master/arcface_comp_for_github.py

**Comment by HuangYG123** [comment]
There are two mistakes in the script.

**Comment by ypw-lbj** [comment]
请问您使用预训练模型了吗？
Have you used the pre training model?
